README.md

```diff
@@ -70,6 +72,16 @@ uds deploy k3d-core-istio-dev:0.15.1
 
 <!-- x-release-please-end -->
 
+Deploy just Istio and Pepr:
+
+<!-- x-release-please-start-version -->
+
+```bash
+uds deploy k3d-core-istio-dev:0.15.1 -p "uds-k3d-dev, init, core-istio"
```

Comment:
Wondering if we should rename this since its no longer just "istio". I don't love changing things around but if we intend to extend this model in the future with other optionality we might want to change the name to `k3d-core-slim-dev` or something? Thoughts?

Reply:
I have no issue renaming it I was trying to introduce as low of an impact of a change as possible and changing the name would be ceasing to publish the existing artifact.

Reply:
Not sure why I didn't see this comment earlier today 😭

Reply:
@mjnagel @jeff-mccoy @Racer159 I updated this PR based on the discussions that were had to move to a "slim dev" package/bundle with keycloak + istio. Is this more or less in line with the desired outcome?